src/jorphan/src/test/java/org/apache/jorphan/exec/TestKeyToolUtils.java

```diff
@@ -41,8 +41,8 @@
     private static String password;
     private static final int validity = 1;
 
-    @BeforeAll
-    public static void setup(@TempDir Path keystoreDir) throws IOException {
+    @BeforeEach
+    public void setup(@TempDir Path keystoreDir) throws IOException {
         keystore = keystoreDir.resolve("dummy-keystore.jks").toFile();
         password = JOrphanUtils.generateRandomAlphanumericPassword(32);
```

Comment:
@ham1 , why is this change?
you are initializing static fields in individual tests.

Please, please, refrain from pushing unrelated changes under a single commit.
Here you replace "junit5->juni5" code under "junit4->junit5" name. That is super-confuse those who review the change.


/cc @pmouawad

Reply:
I don't remember this change, given its single thread and still has static fields, it doesn't look a required change.

Reply:
What might have happened is some global find/replace broke this but I fixed it incorrectly and didn't notice it.